src/cascadia/QueryExtension/ExtensionPalette.idl

```diff
@@ -36,7 +36,16 @@ namespace Microsoft.Terminal.Query.Extension
         Windows.UI.Xaml.Controls.IconElement ResolvedIcon { get; };
 
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> ActiveControlInfoRequested;
-        event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> AIKeyAndEndpointRequested;
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, String> InputSuggestionRequested;
     }
+
+    [default_interface] runtimeclass TerminalContext : IContext
```

Comment:
No need for `[default_interface]`. You can set `IContext` as the default interface by using `[default]` (check out `ActionArgs.idl`, `BaseContentArgs`)